net/node/transactionPool.go

```diff
@@ -206,7 +221,7 @@ func (this *TXNPool) cleanTransactionList(txns []*transaction.Transaction) error
 	if txnsNum != cleaned {
 		log.Info(fmt.Sprintf("The Transactions num Unmatched. Expect %d, got %d .\n", txnsNum, cleaned))
 	}
-	log.Debug(fmt.Sprintf("[cleanProcessTxnPool], Requested %d clean, %d transactions cleaned from localNode.TransPool and remains %d still in TxPool", txnsNum, cleaned, this.GetTransactionCount()))
+	log.Info(fmt.Sprintf("[cleanTransactionList],transaction %d Requested, %d cleaned, Remains %d in TxPool", txnsNum, cleaned, this.GetTransactionCount()))
```

Comment:
So this message will be print for each time the transaction pool be clean up, which means it will be printed with each block be persisted, it is not good idea to print so much of log at info level, but it OK for debug level , if you really care about the transaction pool status at running time, add a extra client interface to get its status will be better